src/modules/nft/data_provider/snapshots/near_enhanced_api__modules__nft__data_provider__nft_info__tests__nfts_by_contract.snap

```diff
@@ -43,12 +37,6 @@ Ok(
                 ),
                 media_hash: None,
                 copies: None,
-                issued_at: Some(
-                    "1646578767534418718",
-                ),
-                expires_at: None,
-                starts_at: None,
-                updated_at: None,
```

Comment:
These are the changes that appeared after [Frol's fix](https://github.com/near/near-enhanced-api-server/commit/6618934d4bf2a3aded91324d5d8238aeb0b2e4db). I can't find time to investigate this. Since the changes are not that bad (we have a theory that no one really needs these fields), and since NFT support is under heavy beta, let's go with this. I or another engineer will definitely have a look at this when we will polish NFT support.